extension/src/shared/reearth/layers/spatialId.ts

```diff
@@ -74,7 +74,7 @@ const getRectangeParamsFromSpace = (space: SpatialIdSpaceData) => {
     ],
   ];
 
-  const height = vertices[0][2];
-  const extrudedHeight = vertices[4][2];
+  const height = vertices[0][2] + space.geoidHeight;
```

Comment:
Ideally, I think the geoid height is a general concept, so we should be able to get it from scene or view. But it's ok for now.

Reply:
I refactored core and Visualizer, now extension will use new APIs to get the geoid height, this property will not appear on space data now.